src/views/common/EditButton.tsx

```diff
@@ -13,7 +13,7 @@ const EditButton = ({
     return (
         <Tooltip
             title="Edit"
-            placement={tooltipPlacement}
+            placement={tooltipPlacement as TooltipProps["placement"]}
```

Comment:
Reusing declarations like this, when inferred types are the norm, is lovely. Just as safe, but far more succinct than the enforced duplication of `javac`.